lib/index.js

```diff
@@ -19,21 +19,27 @@ module.exports = {
 
     return {
       upload: async (file, customParams = {}) => {
-        const path = file.path ? `${file.path}/` : "";
+        const path = file.path ? `${file.path}/` : '';
         const formData = new FormData();
-
-        formData.append("files", file.buffer, {
-          filename: file.hash,
-          contentType: file.mime,
-        });
-        formData.append("files_names", file.hash);
-        formData.append("folder_paths", `${root}/${path}`);
+        const {
+          buffer,
+          hash,
+          mime,
+        } = file;
+
+        const options = {
+          access: 'PUBLIC_NOT_INDEXABLE',
+          overwrite: true,
+          ...customParams,
```

Comment:
Did the other way around, so the options have some default values, but the `customParams` can just override them, what do you think?